src/sql/plot.py

```diff
@@ -519,7 +528,7 @@ def _histogram(table, column, bins, with_=None, conn=None, facet=None):
             floor("{{column}}"/{{bin_size}})*{{bin_size}} as bin,
             count(*) as count
             from "{{table}}"
-            {{filter_query}}
+            {{filter_queries[0]}} {{filter_queries[1]}}
```

Comment:
what if there are multiple filter queries in the future? Is there a better way to loop over the list and add in the template?

Reply:
Sure, I will look into this.

Reply:
I have added a `_filter_aggregate function` that takes in multiple filter queries and generate a single filter. We now add this filter to our template.